src/main/java/com/oltpbenchmark/benchmarks/featurebench/FeatureBenchWorker.java

```diff
@@ -305,17 +305,6 @@ public void tearDown() {
 
     private void executePgStatStatements(List<String> allQueries) throws SQLException {
         String pgStatDDL = "select * from pg_stat_statements;";
-        String PgStatsDir = "ResultsForPgStats";
-        FileUtil.makeDirIfNotExists("results" + "/" + PgStatsDir);
-        String fileForPgStats = PgStatsDir + "/" + workloadName + "_" + TimeUtil.getCurrentTimeString() + ".json";
-        PrintStream ps;
-        try {
-            ps = new PrintStream(FileUtil.joinPath("results", fileForPgStats));
-
-        } catch (FileNotFoundException exc) {
-            throw new RuntimeException(exc);
-        }
-
```

Comment:
trying to understand this code @GargiChakraverty-yb. Are we removing what is being dumped in ResultsForPgStats and ResultsForExplain because it is part of the json?

Reply:
Yes. we are removing the folders because we already have the pg stats and explain for workload queries in output.json

Reply:
Was this discussed with @shaharuk-yb?  I think those folders are still useful given that output.json will get overridden.